sources/AppBundle/Event/Validator/Constraints/PublicTicket.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace AppBundle\Event\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * Class PublicTicket
+ * @Annotation
+ */
+class PublicTicket extends Constraint
+{
+    public $messageNotLoggedIn = 'To order this ticket you must be logged in with a valid membership.';
```

Comment:
dans les contraints précédentes, les messages étaient du type `You must be connected to order this ticket`. 
la construction est différente. Utiliser `You must be connected with a valid membership to order this ticket` ?